spec/core.md

```diff
@@ -18,187 +18,253 @@ DID, after the prefix, is the case-sensitive [[ref: method-specific identifier]]
-just synonyms. Internal processing of DID docs (which uses only the stable form)
-is deterministic and independent of context that a DID owner may not control.
+:::
+
+::: todo
+
+TODO: Consider merging this and the next section and propose that an entity
+SHOULD only publish one "current" `did:webs`, with defined support for redirects
+Copies of a the `did:webs` data should be just that -- copies.
+
+:::
+
+Since an AID is a unique identifier that is inextricably bound to the KEL from
+which it is associated, any `did:webs` DIDs that have the same AID component
+MUST return an equivalent, although not necessarily identical, DIDDoc and KEL.
+Notably, as defined in the [next section](#identifiers-in-a-didwebs-diddoc), the
+`id` field in the DIDDoc will differ based the web location of the DIDDoc. As
+well, different versions of the DIDDoc and KEL may reside in different locations
+depending on the replication capabilities of the controlling entity. If the KELs
+differ for `did:webs` DIDs with the same AID, the smaller KEL MUST be a prefix
+of the larger KEL (e.g., the only difference in the KELs be extra events in one
+of the KELs, not yet reflected in the other). If the KELs diverge from one other
+(e.g., one is not a subset of the other), both the KELs and the DIDs MUST be
+considered invalid.
+
+The web supports a number of ways to redirect users. Please the section on
+[Handling Web Redirections](#) later in this specification.
+
+KERI anticipates the possibility of a duplicitous actor with an AID that forks a
+KEL and shares different versions of the KEL containing different events, such
+as publishing different versions of the KEL on different web servers. The
+verification of the KEL MAY provide mechanisms for detecting such behavior, such
+as KERI witnesses and watchers.
+
+### Handling Web Redirection
+
+A `did:webs` is intended to be a "stable" (long-lasting) identifier that can be
+put into documents, such as verifiable credentials, that are intended to be
+useful for a very long time -- generations. However, the web is not a very
+stable place, and documents are moved around and copied frequently. When two or
+more companies merge, often the web presence of some of the merged entities
+"disappears". It may not be possible to retain a permanent `did:webs` web
+location.
+
+When a `did:webs` moves, its AID does not change. The same KEL is used to verify
+the DIDDoc. Were the AID to change, it would be an altogether new DID,
+unconnected to the first DID. So if a resolver can find a newly named DID that
+uses the same AID, and the KEL verifies the DID, then they have resolved the
+DID. In fact, a `did:webs` could be moved to use another DID method that uses
+the AID for uniqueness and KEL for validity.
+
+The following are the capabilities in `did:webs` to help in the face of
+resolution uncertainty.
+
+- The `did:webs` is bound to its location via an event recording in the KEL, and
+  as the `id` in the DIDDoc.
```

Comment:
I don't think the ìd` property should have the location. But I could be convinced.

Reply:
Let’s leave it for now, and discuss it further.